.github/workflows/build-packages.yml

```diff
@@ -136,6 +136,18 @@ jobs:
           echo "$HOSTKEY" > ~/.ssh/known_hosts
           rsync -4rlptD built_pkgs/* "$RSYNCTARGET"
 
+  check-hashes:
+    needs: build
+    name: Check if hashes were created for all requested targets
+    runs-on: ubuntu-20.04
```

Comment:
might as well do 22.04 for new jobs

Reply:
Do we want to change this to 22.04 before merging?

Reply:
I wanted to create a new PR once this is merged for upgrading all jobs in this workflow to 22.04 to have all of them on the same version.

Reply:
Alright, I'll merge this PR now then :)